pyat/at/load/matfile.py

```diff
@@ -418,21 +421,29 @@ def scan(d):
                     yield convert(k)
                     yield convert(v)
 
-            return "struct({0})".format(", ".join(scan(pdir)))
+            return "struct({})".format(", ".join(scan(pdir)))
```

Comment:
tested save_mat. OK, file is saved and can be loaded in python and in matlab yelding the same lattice.